### ISSUE TITLE
Suggestions: Advanced Peripherals Compatibility

### ISSUE THREAD

**Issue by EndlessEden** [merlinlikethewizard/Mastermine#10]
With PeripheralsPlusOne all but abandoned, and now removed from Curseforge, it seems the best way to get the features from PP1 is via AdvancedPeripherals(https://github.com/Seniorendi/AdvancedPeripherals)

Advanced Peripherals is a port of PeripheralsPlusOne to Minecraft 1.16.x | While currently incomplete, experimenting and planning is possible even at this stage. Please take a look and consider porting your lua to support Advanced Peripherals as well.

**Comment by merlinlikethewizard** [comment]
I agree AdvancedPeripherals is the future! I don't believe chunky turtles are implemented yet though. I asked the creator to look at chunky turtles and they said they've added it to the list, so the minute I hear they're included I'll make sure the program works with them.

In the meantime is there work to do to add support for AdvancedPeripherals? I just assumed that code-wise it wouldn't be all too different, but I'm happy to make changes if needed.

**Comment by ramonGonzEdu** [comment]
> I agree AdvancedPeripherals is the future! I don't believe chunky turtles are implemented yet though. I asked the creator to look at chunky turtles and they said they've added it to the list, so the minute I hear they're included I'll make sure the program works with them.
> 
> In the meantime is there work to do to add support for AdvancedPeripherals? I just assumed that code-wise it wouldn't be all too different, but I'm happy to make changes if needed.

Chunky turtles were added in 0.4a, so they can be implemented now, but you have to make sure to download the alpha build.

**Comment by merlinlikethewizard** [comment]
Hey, would you look at that! Awesome, thank you.
